### PR TITLE
4 - Adding `no-console` and `no-debugger` error rules

### PR DESCRIPTION
**Pull request by dotpointer**
This adds `no-console` and `no-debugger` error rules as requested in #4.

### CODE REVIEW

**Comment by cristinecula** [comment]
@dotpointer `index.js` is the file you need to modify. This PR only adds the rules to this project, not to the shared configuration.

**Review by cristinecula — CHANGES_REQUESTED** [review]
@dotpointer you must add the rules to `index.js`. That is the shared configuration.

**Comment by dotpointer** [comment]
Added the rules to `index.js`, but did not get any indication to remove them from `.eslintrc.json`, please tell me if it should be removed from there.

**Comment by cristinecula** [comment]
Yes, please remove them from `.eslintrc.json`.

**Comment by dotpointer** [comment]
Removed it now.

**Comment by nomego** [comment]
@cristinecula maybe this is the way for no-debugger, https://github.com/microsoft/vscode-eslint/issues/208#issuecomment-323753480

**Comment by cristinecula** [comment]
@nomego what do you think about https://www.npmjs.com/package/eslint-plugin-no-autofix?

**Comment by nomego** [comment]
Sure, works for me.
What about @Neovici/frontend ?

**Comment by JockeCK** [comment]
Do it

File: index.js [hunk 1]
```diff
@@ -115,9 +115,11 @@ module.exports = {
 			}
 		],
 		'no-alert': 'error',
+		"no-autofix/prefer-const": "error",
 		'no-bitwise': 'warn',
-		'no-console': 'off',
+		'no-console': 'error',
```
Comment (cristinecula) [inline index.js]: @dotpointer also replace with `no-autofix/no-console` and `no-autofix/no-debugger`

File: index.js [hunk 2]
```diff
@@ -115,9 +115,11 @@ module.exports = {
 			}
 		],
 		'no-alert': 'error',
+		"no-autofix/prefer-const": "error",
```
Comment (cristinecula) [inline index.js]: Not requested by original issue, but may be useful. @nomego what do you think?

**Review by cristinecula — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (dotpointer) [reply index.js]: Oh, I misunderstood how to use this, it should not be there. :blush:

File: index.js (hunk 1 above)
Comment (dotpointer) [reply index.js]: Aha, OK.

**Comment by dotpointer** [comment]
> Sure, works for me.
> What about @Neovici/frontend ?

What should I change on `cosmoz-frontend`? Same as here? (@cristinecula maybe you know in case nomego is busy.)

**Comment by cristinecula** [comment]
@dotpointer you don't need to change anything in `cosmoz-frontend`, I think @nomego was just requesting feedback from the rest of the team.

File: index.js (hunk 1 above)
Comment (dotpointer) [reply index.js]: Replaced.

File: index.js (hunk 2 above)
Comment (dotpointer) [reply index.js]: Removed this.

Comment (nomego) [reply index.js]: I haven't gotten annoyed by the autofix'es at all yet, and I normally write consts anyway :)
But if anyone's annoyed by it we can try and see if we're more or less annoyed by eslint reporting errors and not fixing them :)

**Review by nomego — APPROVED** [review]

**Pull request merged**